quarkus-opentelemetry-exporter-azure/integration-tests/src/main/resources/application.properties

```diff
@@ -2,4 +2,4 @@
 quarkus.application.name=opentelemetry-exporter-azure-integration-test
 quarkus.application.version=999-SNAPSHOT
 
-quarkus.otel.azure.applicationinsights.connection.string=InstrumentationKey=bla;IngestionEndpoint=http://127.0.0.1:53602/export
+applicationinsights.connection.string=InstrumentationKey=bla;IngestionEndpoint=http://127.0.0.1:53602/export
```

Comment:
We should not do this breacking change...
We should add the new `applicationinsights.connection.string` and use it if `quarkus.otel.azure.applicationinsights.connection.string` is not set... Or the other way around. I'm open to set the property priority that you find best.

Reply:
In the last commit `applicationinsights.connection.string` takes precedence over `quarkus.otel.azure.applicationinsights.connection.string`.  The reason is that the Azure users are used to configure the APPLICATIONINSIGHTS_CONNECTION_STRING environment variable.